build/archive.js

```diff
@@ -229,8 +229,13 @@ function exec(cmdLine, baton, callback) {
     }
 
     child_process.exec(cmdLine, function (err, stdout, stderr) {
-        var err = err || stderr;
-        err && baton ? error(err, baton) : callback((stdout || '').trim(), err);
+        if (!err) { err = stderr; }
+
+        if (err && baton) {
+            error(err, baton);
+        } else {
+            callback((stdout || '').trim(), err);
+        }
```

Comment:
If lint is complaining about it, I guess leave your changes as is (I didn't realize it was, as it doesn't for me. I'll have to figure out why that is).